pkgs/development/tools/frink/default.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs }:
+with pkgs;
```

Comment:
I suspect you mean "lazy" rather than "sluggish".

I don't quite understand what you mean by "detail your inputs", though. Can you please elaborate?

Reply:
Instead of using `{pkgs}: with pkgs; ...` you should use ` { jdk, rlwrap }: ...`.

Reply:
P.S.: not my default lang.